src/common.jl

```diff
@@ -54,6 +56,6 @@ function solve(
         end
     end
     =#
-    DiffEqBase.build_solution(prob,alg,ts,timeseries,du=dus,
```

Comment:
This doesn't seem to be correct. It needs `du= ...`

https://github.com/JuliaDiffEq/DiffEqBase.jl/blob/master/src/solutions/dae_solutions.jl#L18

Reply:
when there was du= build_solution replaced du by nothing, don't sure why, but now it doesn't. That's all the fix, actually